tests/openthread_test/Makefile

```diff
@@ -0,0 +1,47 @@
+APPLICATION = openthread_test
```

Comment:
Since you have this test and in the example above you are basically copy-pasting `examples/posix_sockets` I would prefer just to keep it to this test. Also **tests**/openthread_**test** seems redundant to me.